plugins/crypto/mbedtls/certificategroup.c

```diff
@@ -618,7 +694,9 @@ UA_CertificateGroup_Memorystore(UA_CertificateGroup *certGroup,
     certGroup->removeFromTrustList = MemoryCertStore_removeFromTrustList;
     certGroup->getRejectedList = MemoryCertStore_getRejectedList;
     certGroup->getCertificateCrls = MemoryCertStore_getCertificateCrls;
-    certGroup->verifyCertificate = MemoryCertStore_verifyCertificate;
```

Comment:
Does it make sense to keep the old version around too?
As a shortcut which does all the checks...